Lib/copy.py

```diff
@@ -56,7 +56,7 @@ class Error(Exception):
     pass
 error = Error   # backward compatibility
 
-__all__ = ["Error", "copy", "deepcopy"]
+__all__ = ["Error", "copy", "deepcopy", "replace"]
```

Comment:
Consider adding a regression test. See helper method `support.test__all__` in the test directory.

Reply:
Sure, let me go ahead, good callout.